src/isomorphic/ReactDebugTool.js

```diff
@@ -58,17 +63,40 @@ var ReactDebugTool = {
   onSetState() {
     emitEvent('onSetState');
   },
-  onMountRootComponent(internalInstance) {
-    emitEvent('onMountRootComponent', internalInstance);
+  onSetDisplayName(debugID, displayName) {
+    emitEvent('onSetDisplayName', debugID, displayName);
+  },
+  onSetIsEmpty(debugID, isEmpty) {
+    emitEvent('onSetIsEmpty', debugID, isEmpty);
+  },
+  onSetChildren(debugID, childDebugIDs) {
+    if (!isTopLevelWrapper(debugID)) {
```

Comment:
Can you move this check to where you call `.onSetChildren(...)`?

The reason I want these gone is that my new reconciler won't have any of these limitations but it still wants to target this same ReactDebugTool.